lib/views/general/_frontpage_videos.html.erb

```diff
@@ -42,7 +42,7 @@
     </div>
 
     <div class="cta-button-wrapper">
-      <a class="button" href="https://www.mysociety.org/donate/?utm_medium=link&how-much=10">Donate £10</a>
+      <a class="button" href="<%= donation_url('how-much' => 10, utm_content: 'homepage donate 10', utm_campaign: 'wdtk-homepage-videos') %>">Donate £10</a>
```

Comment:
@ajparsons just want to check the utm tracking here.

Currently this generates the following URL (split for ease of reading):

```
https://www.mysociety.org/donate?
  how-much=10&
  utm_campaign=wdtk-homepage-videos&
  utm_content=homepage+donate+10&
  utm_medium=link&
  utm_source=whatdotheyknow.com
```

I think I'll drop the £ value and change the button text to "Donate now", so that will end up as:

```
https://www.mysociety.org/donate?
  how-much=10&
  utm_campaign=wdtk-homepage-videos&
  utm_content=homepage+donate+now&
  utm_medium=link&
  utm_source=whatdotheyknow.com
```

Is this what you'd expect? Easy to change if there's a better combo.

Reply:
Good on campaigns settings, if dropping "donate £10" drop the how-much because our suggested off-on donation is now lower than that. If keeping it, also add an 'how-often=one-off' parameter.

See https://github.com/mysociety/orgsites/pull/1321 for more info on that.

Can be used in combination for a double button like this (example from the blog)

![image](https://user-images.githubusercontent.com/8157058/225625445-feabeb94-3951-487b-9e22-d44a3ef243fe.png)